### DISCUSSION TITLE
Tailwind CSS IntelliSense not working in VS Code

### DISCUSSION THREAD

**Discussion by Amir0234-afk** [tailwindlabs/tailwindcss#14274] [Help] [answered]
After the recent update of VS Code, the Tailwind CSS IntelliSense extension doesn't work and if it doesn't only a few classes are loaded in the prefix (which I think VS Code itself is loading it since I have used it before) rather than all of them. I wanted to know if there's a problem with my own VS Code or is the extension not compatible with the new update. I will provide screenshots of my project config file and the prefixes that are suggested after writing a part of the code.
![image](https://github.com/user-attachments/assets/98bc033f-71e1-4b47-a336-459f67d90cf9)
![image](https://github.com/user-attachments/assets/75d7c294-fa76-4c93-a1c5-0e47b738c111)
![image](https://github.com/user-attachments/assets/83e39895-816f-42d3-b228-1cea38aba33b)
![image](https://github.com/user-attachments/assets/e03e6098-13e9-4d0e-82f0-b3a4df8b0c3d)

**Comment by philipp-spiess** [comment, marked as answer]
Hey @Amir0234-afk! Thanks for reporting this and sorry for the troubles! We just released v0.12.8 of the VS Code extension with a fix that resolves this issue. If you still have issues, please let us know!
